src/register.jl

```diff
@@ -35,7 +35,7 @@ for the project file in that tree and a hash string for the tree.
 #
 #         # parse the project file
 #         @debug("parse the project file")
-#         project = Pkg.Types.read_project(project_file)
+#         project = Pkg.Types.project_file
```

Comment:
What is `Pkg.Types.project_file`? Seems better to just remove this outcommented function.